apps/test/unit/templates/teacherDashboard/TeacherDashboardNavigationTest.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {BrowserRouter as Router} from 'react-router-dom';
+import {render, screen} from '@testing-library/react';
+import {expect} from '../../../util/reconfiguredChai';
+import TeacherDashboardNavigation from '@cdo/apps/templates/teacherDashboard/TeacherDashboardNavigation.jsx';
+
+const CUSTOM_LINKS = [
+  {
+    label: 'custom link 1',
+    url: '/custom_link_1',
+  },
+  {
+    label: 'custom link 2',
+    url: '/custom_link_2',
+  },
+];
+const AI_TUTOR_LABEL = 'AI Tutor';
+const MANAGE_STUDENTS_LABEL = 'Manage Students';
+
+describe('TeacherDashboardNavigation', () => {
+  it('shows standard links with no AI tutor when no props given', () => {
+    render(
+      <Router>
+        <TeacherDashboardNavigation />
+      </Router>
+    );
+
+    screen.getByText(MANAGE_STUDENTS_LABEL);
```

Comment:
While tedious, do you think it is worth it to have at least one test that checks if each of the standard tabs is shown? Just to make sure in a future PR the `progress` tab doesn't fall off

Reply:
I have a hard time imagining how someone would remove that link accidentally. I think checking one link is sufficient.